.readthedocs.yml

```
@@ -4,12 +4,12 @@ sphinx:
   configuration: doc/conf.py
 
 python:
-    version: 3.8
-    install:
-        - requirements: doc/requirements.txt
-        - method: pip
-          path: .
-    system_packages: true
+  install:
+      - requirements: doc/requirements.txt
+      - method: pip
+        path: .
 
 build:
-    image: latest
+  os: ubuntu-22.04
+  tools:
+    python: "3.8"
```

```suggestion
  os: ubuntu-latest
  tools:
    python: "3.12"
```
?
Or is the idea to "freeze" the doc building.

---

yeah, that's the idea. I briefly tried 3.12 but that failed.

---

:disappointed: